Dockerfile

```diff
@@ -38,6 +40,7 @@ ENV GRAPH_EXP_ENV_ROOT_FOLDER=${GRAPH_EXP_ENV_ROOT_FOLDER:-/explorer}
 
 ENV PROXY_SERVER_HTTP_PORT=${NEPTUNE_NOTEBOOK:+9250}
 ENV PROXY_SERVER_HTTP_PORT=${PROXY_SERVER_HTTP_PORT:-80}
+ENV PROXY_SERVER_HTTPS_PORT=443
```

Comment:
No need to define the HTTPS default port here. It will be defaulted to 443 if not set explicitly